src/components/common/navigation/NavigationRoutes.tsx

```diff
@@ -425,7 +432,9 @@ export function RedirectUserWithSentry({ isFirstLoginUser }) {
     const { pathname } = useLocation()
     useEffect(() => {
         if (pathname && pathname !== '/') Sentry.captureMessage(`redirecting to app-list from ${pathname}`, 'warning')
-        if (isFirstLoginUser) {
+        if(window._env_.K8S_CLIENT){
+          push(`${URLS.RESOURCE_BROWSER}`)
```

Comment:
use const directly. same for all below lines.

Reply:
done